src/main/java/com/angelbroking/smartapi/http/SmartAPIResponseHandler.java

```diff
@@ -1,96 +1,87 @@
 package com.angelbroking.smartapi.http;
 
-import java.io.IOException;
-
+import com.angelbroking.smartapi.SmartConnect;
+import com.angelbroking.smartapi.http.exceptions.*;

```

Comment:
Specific imports

Reply:
Specific imports are done instead of .*;